app/views/events/main_index.html.haml

```diff
@@ -1,9 +1,10 @@
 - title "Events"
 
-.subnavbar
-  .container.mt-4
-    .title Events
-
+.subnavbar.border-bottom
+  .container
+    .row
```

Comment:
We don't need a row if we only have a col-12
Only use a container for this.
Skip the .row and .col-12